fixtures/plans.go

```diff
@@ -17,23 +17,27 @@ type PricingPlanComponent struct {
 type Plans []Plan
 
 type Plan struct {
-	ID         int
-	Name       string
-	PlanGuid   string
-	ValidFrom  time.Time
-	Components []PricingPlanComponent
+	ID          int
+	Name        string
+	PlanGuid    string
+	ValidFrom   time.Time
+	Components  []PricingPlanComponent
+	MemoryInMb  uint
+	StorageInMb uint
```

Comment:
I wonder if one option could be add the `MemoryInMb` and `StorageInMb` to each `PricingPlanComponent` instead to the plan. That would make it easier to later split the usage of a service, for instance: S3 storage usage vs. number of API calls or Bandwidth consumed.

But we do not have this use case right now.